docs/content/develop/generate-providers.md

```diff
@@ -0,0 +1,113 @@
+   ```bash
+   make provider VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google" PRODUCT=[PRODUCT_NAME]
+   ```
+    Where [PRODUCT_NAME] is one of the folder names in
+    https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/products.
+  
+    For example, if your product is `bigqueryanalyticshub`, the command would be
+    the following:
+
+     ```bash
+     make provider VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google" PRODUCT=bigqueryanalyticshub
+     ```
+
+1. Generate changes for the `google-beta` provider:
+   ```bash
+   make provider VERSION=beta OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta" PRODUCT=[PRODUCT_NAME]
+   ```
+
+    Where [PRODUCT_NAME] is one of the folder names in https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/products.
+   
+    For example, if your product name is `bigqueryanalyticshub`, the command would be the following:
+
+     ```bash
+     make provider VERSION=beta OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta" PRODUCT=bigqueryanalyticshub
+     ```
+ 
+1. Confirm that the expected changes were generated:
+   ```bash
+   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
+   git diff -U0
+   cd $GOPATH/src/github.com/hashicorp/terraform-provider-google-beta
+   git diff -U0
+   ```
+
+    In both cases, the changes should include:
+
+    ```diff
+    diff --git a/google/services/bigqueryanalyticshub/resource_bigquery_analytics_hudiff --git a/google/services/bigqueryanalyticshub/resource_bigquery_analytics_hub_data_exchange.go b/google/services/bigqueryanalyticshub/resource_bigquery_analytics_hub_data_exchange.go
+    --- a/google/services/bigqueryanalyticshub/resource_bigquery_analytics_hub_data_exchange.go
+    +++ b/google/services/bigqueryanalyticshub/resource_bigquery_analytics_hub_data_exchange.go
+    @@ -66 +66 @@ func ResourceBigqueryAnalyticsHubDataExchange() *schema.Resource {
+    -                               Description: `Human-readable display name of the data exchange. The display name must contain only Unicode letters, numbers (0-9), underscores (_), dashes (-), spaces ( ), and must not start or end with spaces.`,
+    +                               Description: `UPDATED_DESCRIPTION`,
+    diff --git a/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown b/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
+    --- a/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
+    +++ b/website/docs/r/bigquery_analytics_hub_data_exchange.html.markdown
+    @@ -63 +63 @@ The following arguments are supported:
+    -  Human-readable display name of the data exchange. The display name must contain only Unicode letters, numbers (0-9), underscores (_), dashes (-), spaces ( ), and must not start or end with spaces.
+    +  UPDATED_DESCRIPTION
+    ```
```

Comment:
As written, this assumes that users have made a specific change - but that part of the quickstart is no longer here. Maybe we should just delete this section?

```suggestion
```